packages/logging/service.go

```diff
@@ -166,6 +174,16 @@ func (s *loggingService) setLogLevel(ctx context.Context, doguName string, l Log
 
 	logrus.Debugf("written new log level %s for dogu %s", l.String(), doguName)
 
+	deployment, err := s.deploymentGetter.Get(ctx, doguName, metav1.GetOptions{})
+	if err != nil {
+		return false, fmt.Errorf("could not get deployment to check status of dogu: %w", err)
+	}
+
+	if deployment.Status.ReadyReplicas == 0 {
```

Comment:
Is "ReadyReplicas" really the indicator, that the dogu ist stopped?
It could happen, that the dogu has errors and has no "ready" replicas, but is started (e.g. the desired state is, that it should have more than one replica).

Maybe "deployment.Spec.Replicas" is better fitting, because that should be the desired state.